src/python/WMCore/MicroService/MSRuleCleaner/MSRuleCleanerWflow.py

```diff
@@ -216,7 +219,8 @@ def docSchema(self):
             ('RequestTransition', [], list),
             ('IncludeParents', False, bool),
             ('InputDataset', None, (bytes, str)),
-            ('ParentDataset', None, (bytes, str))]
+            ('ParentDataset', None, (bytes, str)),
+            ('StatusAdvanceExpiredMsg', "", (bytes, str))]
```

Comment:
Even though it's not going to be consistent with the rest of the schema, can you please decide which data type it's supposed to be (now that we are fully in python3).

Reply:
agreed